tests/c_languages/CPPCleanBearTest.py

```diff
@@ -17,6 +18,39 @@
 """
 
 
+test_file = """
+#include "CPPCleanBearHeaderTest1.h"
+#include "CPPCleanBearHeaderTest2.h"
+
+int main() {
+    cppcleanbearfunction1();
+    cppcleanbearfunction2();
+    return 0;
+}
+"""
+
+
 CPPCleanBearTest = verify_local_bear(CPPCleanBear,
                                      valid_files=(good_file,),
-                                     invalid_files=(bad_file,))
+                                     invalid_files=(bad_file,),
+                                     tempfile_kwargs={'suffix': '.cpp'})
+
+CPPCleanBearNoIncludePathTest = verify_local_bear(
+    CPPCleanBear,
+    valid_files=(),
+    invalid_files=(test_file,),
+    tempfile_kwargs={'suffix': '.cpp'}
+    )
+
+CPPCleanBearValidIncludePathTest = verify_local_bear(
+    CPPCleanBear,
+    valid_files=(test_file,),
+    invalid_files=(),
+    settings={'include_paths':  os.path.join(os.path.dirname(__file__),
+                                             'test_files',
+                                             'testCPPHeader1')+','
+              + os.path.join(os.path.dirname(__file__),
+                             'test_files',
+                             'testCPPHeader2')},
```

Comment:
please write a function for that (best is you call it `get_testfile_name` because other tests do the same), so you don't have to repeat yourself:
```
'include_paths': get_testfile_name('testCPPHeader1') + ',' + get_testfile_name(...)
```
Or call it `get_testdirectory_name` or so.